lib/index.js

```diff
@@ -37,7 +37,7 @@ function setupTracker(matomoUrl, token, spec) {
 
   return {
     track(status, req) {
-      if (process.env.NODE_ENV !== "production") {
+      if (process.NODE_ENV !== "production") {
```

Comment:
Are you sure this change is correct? I’ve never seen `process.NODE_ENV` before, while `process.env.NODE_ENV` is common.

Reply:
yeah, well, oops 😬 my bad, thanks a lot for spotting it!